specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2021-10-01/imds.json

```diff
@@ -961,15 +965,8 @@
       "name": "api-version",
       "in": "query",
       "type": "string",
-      "enum": [
-        "2021-10-01"
-      ],
-      "description": "This is the API version to use.",
-      "required": true,
-      "x-ms-enum": {
-        "name": "ApiVersion",
-        "modelAsString": true
-      }
+      "description": "Version of the API to be used with the client request.",
```

Comment:
Changed this to a string to avoid cross-breaking change detection triggering with each version bump. Aligns with what other azure services does in this repository